openedx/core/lib/django_require/staticstorage.py

```diff
@@ -4,9 +4,10 @@
 
 from pipeline.storage import PipelineCachedStorage
 from require.storage import OptimizedFilesMixin
+from django_pipeline_forgiving.storages import PipelineForgivingStorage
 
 
-class OptimizedCachedRequireJsStorage(OptimizedFilesMixin, PipelineCachedStorage):
+class OptimizedCachedRequireJsStorage(OptimizedFilesMixin, PipelineForgivingStorage):
```

Comment:
The errors aren't logged.  Instead of getting a hashed name, you just get the unhashed name.  So, I think, short of the collection process itself breaking, if the file isn't there, it's not going to be there at all.  Like, this would only ever come up if you had imports in CSS because that's the only time `django-pipeline` is trying to touch a file it didn't get from a directory listing.